Makefile

```diff
@@ -86,6 +86,12 @@ fallthrough: submodules
 	@echo Initial setup complete. Running make again . . .
 	@make
 
+# TODO(negz): Merge tags support back into https://github.com/upbound/build
+generate:
+	@$(INFO) go generate -tags generate $(PLATFORM)
```

Comment:
one of the principles that @bassam likes to adhere to, is to download go modules inside `$(WORK_DIR)`. For this to happen, I exported `GOPATH` to a subfolder there. 
https://github.com/upbound/build/pull/74/files#diff-a01ea346de0fbd4a8a8cfccb3b0d7793R143

My suspicion is that the above command will download the required packages in the default GOPATH (which is `$(HOME)/go/mod`), breaking that principle.

Reply:
I raised https://github.com/upbound/build/pull/78 to make the `go generate` changes upstream, and have confirmed tooling is downloaded to `$WORK_DIR`.